extensions/ql-vscode/src/remote-queries/variant-analysis-manager.ts

```diff
@@ -72,6 +72,17 @@ export class VariantAnalysisManager extends DisposableObject implements VariantA
     }
   }
 
+  public async removeVariantAnalysis(variantAnalysis: VariantAnalysis) {
+    this.variantAnalysisResultsManager.removeAnalysesResults(variantAnalysis);
```

Comment:
I think this should be named `removeAnalysisResults` as it's removing multiple results for only one analysis?

Reply:
I started suggesting the change but it has a couple of occurrences so it's easier to do it in the editor